helm/cluster/files/apps/k8s-audit-metrics.yaml

```diff
@@ -3,7 +3,8 @@ configKey: k8sAuditMetrics
 clusterValues:
   configMap: true
   secret: false
-dependsOn: kyverno
+dependsOnHelmRelease: true
+dependsOn: kyverno, cilium, prometheus-operator-crd
```

Comment:
This app in fact depends on Cilium and Prometheus Operator CRD. With switch from Chart Operator to Flux it fails without these dependencies set.